app/main.py

```diff
@@ -1,6 +1,17 @@
-from typing import Callable
+from typing import Callable, Any
+import functools
```

Comment:
`import` should precede `from ... import`, place your import in the right order.

Reply:
Place your imports in the right order according to the previous comment.